hack/jenkins/test-dockerized.sh

```diff
@@ -19,6 +19,13 @@ set -o nounset
 set -o pipefail
 set -o xtrace
 
+function _retry {
```

Comment:
minor nit: I don't think we use underscore-prefixed function names anywhere else in the repo. is there something you're worried about shadowing?